src/buildbot.py

```diff
@@ -48,6 +57,12 @@ def BuildNumber():
   return BUILDBOT_BUILDNUMBER
 
 
+def IsEmscriptenReleasesBot():
+  """Return true if running on the emscripten-releases builders,
+     False otherwise."""
+  return BUILDBOT_MASTERNAME is EMSCRIPTEN_RELEASES_BOT
```

Comment:
let's use `==` here for consistency with the rest of this repo.

Reply:
Done.